templates/component-mapper/src/src/files/radio.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Radio = () => 'Radio';
```

Comment:
Again we can use some basic implementation to showcase what props and how to use them in order to get this working. Mainly the onChange is important